Tests/test_GenBank.py

```diff
@@ -7817,6 +7817,7 @@ def test_qualifier_escaping_write(self):
         f2 = record.features[1]
         f1.qualifiers["note"][0] = '"Should" now "be" escaped in "file"'
         f2.qualifiers["note"][0] = '"Should also be escaped in file"'
+        genbank_out = "GenBank/qualifier_escaping_write.test.gb"
```

Comment:
Good catch that this was overwriting a test file, but better to output this to a temp location?

Reply:
done